cmd/common/registry.go

```diff
@@ -259,12 +259,11 @@ func buildCommandFromGadget(
 			// Before running the gadget, we need to get the gadget info to create the
 			// parser based on it
 			if isRunGadget {
-				if len(args) == 0 {
-					if showHelp, _ := cmd.Flags().GetBool("help"); showHelp {
-						additionalMessage := "Specify the gadget image to get more information about it"
-						cmd.Long = fmt.Sprintf("%s\n\n%s", cmd.Short, additionalMessage)
-						return cmd.Help()
-					}
+				showHelp, _ := cmd.Flags().GetBool("help")
+				if len(args) == 0 || showHelp {
+					additionalMessage := "Specify the gadget image to get more information about it"
```

Comment:
This additional message will now be printed if there are no args, but it is only true when `-help` is given.
I think we need another message for when only `len(args) == 0`?